### PR TITLE
Fileselect docs

### PR DESCRIPTION
**Pull request by ntacheva**
## New Component Checklist

- [x] - Overview article
    - [x] General information on the purpose of the component.
    - [x] Instruction on how to add the component to a razor component.
    - [x] Basic configuration of the component - code snippet with a scenario that is both basic, and meaningful.
    - [x] Provide information on the component's reference.
    - [x] List the available features for the component and/or provide links (slugs) for the articles that explain them in details. For simpler components listing parameters might be enough, for more complex ones sections with a sentence or two that links corresponding articles might be more suitable.
- [x] - Feature/component specific articles - those articles explain the component specific features, which do not suit in any of the before-mentioned articles. Examples of such articles are the [Navigator](https://docs.telerik.com/blazor-ui/components/stockchart/navigator) for the Stock Chart, the [Indeterminate](https://docs.telerik.com/blazor-ui/components/checkbox/indeterminate-state) state for the CheckBox and others. 
- [x] - Events article - applicable if the component expose events. 
- [x] Link from the Overview components list

### CODE REVIEW

File: components/fileselect/events.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+@[template](/_contentTemplates/common/general-info.md#event-callback-can-be-async)
+
+
+## OnRemove
+
+The `OnRemove` fires when a file has been removed from the list of selected files (by clicking the `x` icon or pressing `Del` key). Contains the removed fileInfo object.
+
+The event handler receives a `FileSelectEventArgs` object. Its `Files` field is a collection of `FileSelectFileInfo` objects. As the FileSelect component allows deleting one item at a time, the collection contains only one `FileSelectFileInfo` object (the deleted one) and it has the following fields:
+
+* `Id`- `string` - the unique identifier of the file.
```
Comment (svdimitr) [inline components/fileselect/events.md:90]: Maybe we should move all of these to tables, but if you do not have the time to do that quickly now, we should do it after the release

**Review by svdimitr — CHANGES_REQUESTED** [review]

**Comment by svdimitr** [comment]
One additional thing might be a FileSelect vs Upload section (or article). We might think about it now, or in future, but if(future){logInGitHub}

File: components/fileselect/events.md (hunk 1 above)
Comment (ntacheva) [reply components/fileselect/events.md:90]: Done

File: components/fileselect/overview.md [hunk 2]
```diff
@@ -32,9 +54,9 @@ The Telerik FileSelect for Blazor provides a [Stream](https://docs.microsoft.com
 
 The FileSelect component provides the following key features:
```
[the remark this answers is not in the document]
Comment (ntacheva) [reply components/fileselect/overview.md:55]: Done

File: components/fileselect/overview.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+tags: telerik,blazor,fileselect,async,overview
+published: True
+position: 0
+---
+
+# FileSelect Overview
+
+The <a href = "https://www.telerik.com/blazor-ui/fileselect" target="_blank">Blazor FileSelect component</a> helps users select single or multiple files from their local file systems. It provides UI for selecting the files and allows the application logic to handle the actual upload as desired. The component is especially useful when you want full control over the process of creating the server requests and sent forms.
+
+The Telerik FileSelect for Blazor provides a [Stream](https://docs.microsoft.com/en-us/dotnet/api/system.io.stream?view=net-6.0) for each selected file, so that you can save it to memory, a file system or other destination.
```
[the remark this answers is not in the document]
Comment (ntacheva) [reply components/fileselect/overview.md:15]: Updated

File: components/fileselect/validation.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+description: Validate the selected files in the FileSelect for Blazor.
+slug: fileselect-validation
+tags: telerik,blazor,fileselect,async,validate,validation
+published: true
+position: 10
+---
+
+# FileSelect - Selected Files Validation
+
+Files must be validated when uploading, and the process has two parts:
```
[the remark this answers is not in the document]
Comment (ntacheva) [reply components/fileselect/validation.md:13]: Updated

**Comment by ntacheva** [comment]
> FileSelect vs Upload section (or article)

Indeed, I logged a task to do after the release if there is not enough time before that.

**Pull request merged**